src/webview/helm-chart/app/helmModal.tsx

```diff
@@ -0,0 +1,209 @@
+                        alignItems='flex-start'
+                        marginBottom={1}
+                    >
+                        <HelmListItem helmChart={props.helmChart} selectedVersion={selectedVersion} />
+                        <IconButton onClick={() => {
+                            props.closeModal()
+                        }}>
+                            <Close color='textSecondary' />
+                        </IconButton>
+                    </Stack>
+                    <TextField fullWidth
+                        id='name'
+                        variant='outlined'
+                        label='Name'
+                        value={installName}
+                        error={!isInstallNameFieldValid}
+                        helperText={installNameErrorMessage}
+                        onChange={(e) => {
+                            setStatus(false);
+                            setInstallError(false);
+                            setInstallMsg('');
+                            props.closeModal
+                            VSCodeMessage.postMessage({
+                                action: 'validateName',
+                                data: e.target.value
+                            });
+                            setInstallame(e.target.value);
+                        }} />
+                    <FormControl fullWidth>
+                        <InputLabel id='version'>Versions</InputLabel>
+                        <Select
+                            value={selectedVersion?.version}
+                            onChange={(event) => {
+                                const chartVersion = props.helmChart.chartVersions.filter((chart) => chart.version === event.target.value).pop();
+                                setSelectedVersion(chartVersion);
+                            }}
+                            onClick={(_e) => {
+                                setInteracted(true);
+                            }}
+                            disabled={versions.length < 2}
+                            error={isError}
+                            sx={{ flexGrow: '1' }}
+                            label='Versions'
+                            labelId='version'
+                        >
+                            {versions.map((chartVersion) => {
+                                return (
+                                    <MenuItem value={chartVersion.version} key={chartVersion.version}>
+                                        {chartVersion.version}
+                                    </MenuItem>
+                                );
+                            })}
+                        </Select>
+                        <Stack direction='row' justifyContent='space-between'>
+                            <FormHelperText error={isError}>{helperText}</FormHelperText>
+                            <Stack direction='row' marginTop={1} spacing={2}>
+                                <LoadingButton
```

Comment:
I can click the install button when I first open the modal, before I enter any text into the name field. I think the button should be disabled in this case.